pkgs/os-specific/linux/pam_gnupg/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchgit, autoreconfHook, gnupg, pam } :
+
+stdenv.mkDerivation rec {
+  pname = "pam_gnupg";
+  version = "unstable-2019-12-06";
+
+  src = fetchgit {
+    url = https://github.com/cruegge/pam-gnupg;
+    rev = "fbd75b720877e4cf94e852ce7e2b811feb330bb5";
+    sha256 = "0kqn6xb85jfmhvvbd2lasnci46p2pcwy0wq233za9h7xwfr49f7d";
+  };
+
+  nativeBuildInputs = [ autoreconfHook ];
+  buildInputs = [ gnupg pam ];
+
+  preAutoreconf = ''
+    mkdir m4
+  '';
```

Comment:
Is this really necessary?

Reply:
It silenced a warning during the build phase, but I removed it because it doesn't cause the build to fail.